.github/workflows/cwf-operator.yml

```diff
@@ -0,0 +1,100 @@
+name: cwf-operator
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'cwf/**'
+      - 'k8s/**'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'cwf/**'
+      - 'k8s/**'
+jobs:
+
+########## CWF-PRECOMMIT ##########
+
+  cwf-operator-precommit:
+
+    runs-on: ubuntu-latest
+
+    env:
+      GO111MODULE: on
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Export Magma root path
+      run: echo "MAGMA_ROOT=${PWD}" >> $GITHUB_ENV
+    - name: Run golang_before_install.sh script
+      run: ./circleci/golang_before_install.sh
+    - name: Move into cwf dir
+      run: cd ${MAGMA_ROOT}/cwf/k8s/cwf_operator
+    - name: Run go mod download with retry
+      uses: nick-invision/retry@v2
+      with:
+        command: cd ${MAGMA_ROOT}/cwf/k8s/cwf_operator && go mod download
+        timeout_minutes: 10
+    - name: Run precommit
+      run: |
+          make -C ${MAGMA_ROOT}/cwf/k8s/cwf_operator precommit
+#    - magma_slack_notify
+
+########## CWF-OPERATOR-BUILD ##########
+
+  cwf-operator-build:
+
+    needs: cwf-operator-precommit
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Export Magma root path
+      run: |
+          echo "MAGMA_ROOT=${PWD}" >> $GITHUB_ENV
+    - name: Build the docker-compose stack
+      run: |
+           cd ${MAGMA_ROOT}/cwf/k8s/cwf_operator/docker
+           docker-compose up -d
+    - name: Run docker compose build
+      run: |
+           cd ${MAGMA_ROOT}/cwf/k8s/cwf_operator/docker
+           DOCKER_REGISTRY=cwf_ docker-compose build --parallel
+    - name: Login to Docker registry
+      uses: docker/login-action@v1
+      with:
+          registry: ${{ DOCKER_MAGMA_REGISTRY }}
+          username: ${{ USERNAME }}
+          password: ${{ PASSWORD }}
+    - name: Build and push
+      uses: docker/build-push-action@v2
```

Comment:
Not sure if this is going to work.. I will be able to test it once we have the secrets in place but I am thinking if we can only use the push part of this action and not the build since we are already doing the build in line 64

If not, I will need to figure how to configure this action to run the docker compose command like we need to.